src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+
+JavaScript will always apply the logical NOT operator, ``!``, first. It then applies the arithmetic operators (``**`` is highest, then ``*`` and ``/``, then ``+`` and ``-``). Next comes the comparison operators. Finally, the logical AND and OR are applied last. 
+
+This means that the expression ``x * 5 >= 10 && y - 6 <= 20`` will be evaluated so as to first perform the arithmetic and then check the relationships. The ``&&`` evaluation will be done last. The order of evaluation is the same as if we were to use parentheses to group, as follows:
+
+.. sourcecode:: js
+
+   ((x * 5) >= 10) && ((y - 6) <= 20)
+
+While parentheses make expression more readable, they are not strictly necessary due to default operator precedence.
+
+The following table lists operators in order of precedence, from highest (applied first) to lowest (applied last). A complete table for the entire language can be found in the `MDN JavaScript Documentation <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table>`_.
+
+.. list-table:: Operator Precedence
+   :widths: auto
+   :header-rows: 1
+
+   * - Precedence
+     - Category
+     - Operators
+   * - (highest)
+     - Logical NOT
+     - ``!``
+   * -
+     - Exponentiation
+     - ``**``
+   * -
+     - Multiplication and division
+     - ``*``, ``/``, ``%``
+   * -
+     - Addition and subtraction
+     - ``+``, ``-``
+   * -
+     - Comparison
+     - ``<=``, ``>=``, ``>``, ``<``
+   * -
+     - Equality
+     - ``===``, ``!==``, ``==``, ``!=``
+   * -
+     - Logical AND
+     - ``&&``
+   * - (lowest)
+     - Logical OR
+     - ``||``
+
+Truth Tables
+------------
+
+.. index:: ! truth table
+
+**Truth tables** help us understand how logical operators work by calculating all of the possible return values of a boolean expression. Let's look at the truth table for ``&&``, which displays assumes we have two boolean expressions, A and B, joined by ``&&``.
+
+.. admonition:: Question
+
+   .. list-table:: Truth Table for ``&&``
+      :widths: auto
+      :header-rows: 1
+
+      * - A
+        - B
+        - A ``&&`` B
+      * - ``true``
+        - ``true``
+        - ``true``
+      * - ``true``
+        - ``false``
+        - ``false``
+      * - ``false``
+        - ``true``
+        - ``false``
+      * - ``false``
+        - ``false``
+        - ``false``
+
+Consider the first row of the truth table. This row states that if A is true and B is true, then A && B is true. This is a fact, regardless of what boolean expressions A and B might actually be. The two middle rows demonstrate that if either A or B is false, then A && B is false. (If this idea is hard to grasp, try substituting actual expressions for A and B.)
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   Complete the truth table for ``||``.
```

Comment:
Formatting: "Truth table for ``||``" is redundant after the instruction "Complete the truth table for ``||``."

Reply:
Per previous discussion about table titles, I'm going to leave it but rephrase both the title and instructions to reduce redundancy.